build/workspace.bzl

```diff
@@ -122,7 +122,7 @@ def debian_image_dependencies():
             digest = _digest(_DEBIAN_IPTABLES_DIGEST, arch),
             registry = "k8s.gcr.io",
             repository = "debian-iptables",
-            tag = "v12.0.1",  # ignored, but kept here for documentation
+            tag = "v12.0.3",  # ignored, but kept here for documentation
```

Comment:
as the comment says this is ignored. what is not ignored is the digests above 😅

Reply:
@BenTheElder So if you look at https://github.com/kubernetes/kubernetes/commit/aff4d7ff1c851b2d1c8af9cd3c30fdcd51eb09f3 the digests have not been updated.

Also note that i can't update the digest until we mint those 2 images :)

Reply:
I wouldn't expect the versions to update without the digests, we shouldn't be "consuming" images that don't exist.

Reply:
Going to drop `workspace.bzl` from this PR. Will just update the images. then use the promoter to get them to the official repo. then will file a follow up PR for updating the digest etc.